modules/ROOT/pages/modularizing-your-configuration-files-for-team-development.adoc

```diff
@@ -4,51 +4,44 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: anypoint studio, team, shared projects
 
-Though it may seem convenient to keep all your Mule configuration in one place,
-the reality is that a gigantic XML file quickly becomes unmanageable. This is why
- it is recommended to split monolithic configurations into several files and leverage
-  Mule's capacity to load multiple configuration files at application start-up time.
- Moreover, splitting configurations into multiple fragments encourages re-use across teams.
+Though it seems convenient to keep your Mule configurations in one file,
```

Comment:
RE "Though it seems convenient" : How can we tell readers what seems convenient to them? "Though it might seem convenient"? Or maybe just reword the whole sentence.

Reply:
OK